internal/jimm/identity/identity.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2025 Canonical.
+
+package identity
+
+import (
+	"context"
+
+	"github.com/canonical/jimm/v3/internal/common/pagination"
+	"github.com/canonical/jimm/v3/internal/db"
+	"github.com/canonical/jimm/v3/internal/dbmodel"
+	"github.com/canonical/jimm/v3/internal/errors"
+	"github.com/canonical/jimm/v3/internal/openfga"
+)
+
+// identityManager provides a means to manage identities within JIMM.
+type identityManager struct {
```

Comment:
i would suggest exporting the `identityManager` type

Reply:
As discussed, I think we should definitely do this but in a separate piece of work.

Reply:
please create a jira ticket to at least discuss the issue if not to address it